Extensions/mutualchecker.js

```diff
@@ -17,6 +17,11 @@ XKit.extensions.mutualchecker = new Object({
 			default: "",
 			value: "",
 			type: "blog"
+		},
+		"put_in_front": {
+			text: "Place mutual icon in front",
```

Comment:
minor nitpick but I feel like this option could be more descriptive - "Place mutual icon before, not after usernames" or something similar